munit-hedgehog/shared/src/main/scala/munit/HedgehogSuite.scala

```diff
@@ -0,0 +1,115 @@
+package munit
+
+import hedgehog._
+import hedgehog.core.{PropertyConfig, PropertyT, Seed}
+import hedgehog.predef.Monad
+
+import munit.internal.Compat._
+
+/**
+ * Provides the ability to write property based tests using the Hedgehog library.
+ *
+ * Properties are defined by calling one of the various `property` or `propertyF`
+ * methods and passing a `PropertyT[Result]` or `PropertyT[F[Result]]` respectively.
+ * For example:
+ * {{{
+ * property("additive identity") {
+ *   Gen.int(Range.linearFrom(0, Int.MinValue, Int.MaxValue)).forAll.map { n =>
+ *     assertEquals(n + 0, n)
+ *   }
+ * }
+ * }}}
+ *
+ * Note both Hedgehog `Result` and munit assertions are supported.
+ */
+trait HedgehogSuite extends FunSuite {
+
+  /**
+   * Provides the default configuration for running property tests.
+   * This can be overriden to change the default or can be changed on
+   * a test-by-test basis by using an overload of `property` and `propertyF`.
+   */
+  lazy val propertyConfig: PropertyConfig = PropertyConfig.default
+  lazy val propertySeed: Long = System.currentTimeMillis()
```

Comment:
Not a huge deal, but I would like to standardize these with respect to the scalacheck module. A few differences I can spot:

- we use `protected` in `ScalaCheckSuite` (maybe redundant, but I think it makes sense)
- we use `def` for the config, to allow overriding with anything (`def`,  `val`, `lazy val`) instead of forcing to use `val` in the override
- we prefix the variables using `scalaCheck` as in `scalaCheckTestParameters`. Since these modules are defined as `trait` I wanted to maintain some kind of namespace to avoid potential clashes when we add more modules that can be mixed and matched.

I'm fine to standardize either way, as long as it's consistent.

What do you think @mpilquist?

Reply:
👍 on `protected` and prefixing names. Re: def versus val, I used lazy val in order to guarantee the same seed is used in all tests in a suite. I'm okay with using a def instead though. No strong opinion.

Reply:
> def versus val, I used lazy val in order to guarantee the same seed is used in all tests in a suite

👍 makes sense if that's the intended semantics. Does the same reasoning apply to the config?